chart/index.yaml

```diff
@@ -13,7 +23,7 @@ entries:
     version: 0.3.2
   - apiVersion: v2
     appVersion: 0.4.0
-    created: "2020-06-07T15:18:14.495561112+02:00"
+    created: "2020-06-18T11:48:46.209641+02:00"
```

Comment:
Why are there updates for the other versions ? Only the latest version which is "created" should be shown.

Reply:
@waveywaves we discovered with @tomaszsek a bug which does that. We will fix this ASAP.